stories/features/rewards/settingsMobile/contributeBoxMobile.tsx

```diff
@@ -71,26 +75,74 @@ class ContributeBoxMobile extends React.Component<Props, State> {
         <Grid columns={1} customStyle={{ maxWidth: '270px', margin: '0 auto' }}>
```

Comment:
Couldn't figure out where the max-width was coming from! If this is a storybook only thing than no sweat but we're doing this in desktop and it should be omitted in both places (I can file a followup issue). Causes the settings to look like they are floating in the middle of the card. Something like this: https://d.pr/i/a1ZJ69

Reply:
Yes please file a follow up, we can get that fixed :)